pvlib/test/test_tracking.py

```diff
@@ -188,12 +188,31 @@ def test_SingleAxisTracker_tracking():
 
     tracker_data = system.singleaxis(apparent_zenith, apparent_azimuth)
     
-    expect = pd.DataFrame({'aoi': 7.286245, 'surface_azimuth': 37.3427,
+    expect = pd.DataFrame({'aoi': 7.286245, 'surface_azimuth': 142.65730 ,
                            'surface_tilt': 35.98741, 'tracker_theta': -20.88121},
                            index=[0], dtype=np.float64)
     
     assert_frame_equal(expect, tracker_data)
 
+    ### results calculated using PVsyst
+    pvsyst_solar_azimuth = 7.1609
+    pvsyst_solar_height = 27.315
+    pvsyst_axis_tilt = 20.
+    pvsyst_axis_azimuth = 20.
+    system.backtrack = False
+    system.max_angle = 60.
+    system.axis_tilt = pvsyst_axis_tilt
+    system.axis_azimuth = 180+pvsyst_axis_azimuth
```

Comment:
It will be more clear what's actually being tested if you make these assignments when the object is created.